### DISCUSSION TITLE
Change Alert type

### DISCUSSION THREAD

**Discussion by cyberaa** [thuliteio/doks#582] [Q&A] [answered]
Is it possible to change alert types? 

Some themes(and bootstrap) allow you to use several type of alert boxes like:

- **info**
- **warning**
- **danger**

This would change the background color of the alert box to blue/yellow/red.

**Example:**

```
{{< alert warning icon="❗" text="This is a warning" />}}
```

Thanks

**Comment by h-enk** [comment, marked as answer]
Interesting, not yet supported.

Will make this an option, planned for next release

**Reply by h-enk** [reply]
Implemented: https://github.com/h-enk/doks/issues/583
